lib/internal/abort_controller.js

```diff
@@ -357,6 +362,21 @@ function transferableAbortController() {
   return AbortController[kMakeTransferable]();
 }
 
+/**
+ * @param {AbortSignal} signal
+ * @param {any} resource
+ * @returns {Promise<void>}
+ */
+async function aborted(signal, resource) {
+  abortSignalValidator(signal, 'signal');
+  validateObject(resource, 'resource', { nullable: false, allowFunction: true, allowArray: true });
```

Comment:
Should arrays and functions be allowed here? @benjamingr

Reply:
Yes, a resource can be wrapped in a function or array I guess